annotations/src/main/java/module-info.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2015 The Error Prone Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+open module com.google.errorprone.annotation {
+  requires java.base;
+  requires java.compiler;
+  exports com.google.errorprone.annotations;
+  exports com.google.errorprone.annotations.concurrent;
+}
```

Comment:
Expressed as an `open module` to enable reflective introspection of annotations.